HOC/MotionWrapperHeadingThree.tsx

```diff
@@ -5,25 +5,26 @@ import { motion } from 'framer-motion';
 import { fadeIn } from '@/utils/motion';
 import { MotionHeadingTwoProps } from '@/types';
 
-const MotionWrapperHeadingTwo = ({
+const MotionWrapperHeadingThree = ({
   children,
   direction,
   type,
   delay,
   duration,
   additionalStyles,
+  once = true,
 }: MotionHeadingTwoProps) => {
   return (
-    <motion.h2
+    <motion.h3
       variants={fadeIn(direction, type, delay, duration)}
-      viewport={{ once: true, amount: 0.1 }}
+      viewport={{ once, amount: 0.1 }}
       initial="hidden"
       whileInView="show"
-      className={`${additionalStyles} text-[2.25rem] font-bold leading-[2.6rem] tracking-[-0.36px] md:text-[3rem] md:leading-[3.45rem] md:tracking-[-0.48px]`}
+      className={`${additionalStyles}`}
```

Comment:
```suggestion
      className={additionalStyles}
```
Same thing here. No need for a template literal.